bool found_rail
float64 rail_angle
float64 rail_pixel_width
float64 rail_pixel_x
float64 rail_pixel_y
